versions

```diff
@@ -1,5 +1,8 @@
-BACKPORTS_RELEASE_TAG="UBUNTU_2204_OEM_6043_PRERELEASE_220805.0"
-DII_KERNEL_TAG="DII_6043_prerelease"
-DII_KERNEL_HEAD="5bb330bd8e370e"
-BASE_KERNEL_NAME_20.04="5.14.0-1047"
-BASE_KERNEL_NAME_22.04="5.17.0-1014"
+BACKPORTS_RELEASE_TAG="I915_6213_PRERELEASE_220914.2"
+DII_KERNEL_TAG="PROD_ATSM_6213.0.1"
+DII_KERNEL_HEAD="e8a6ee40e1a3ff"
+UBUNTU_GENERIC_KERNEL_VERSION="5.15.0-50"
+UBUNTU_OEM_22.04_KERNEL_VERSION="5.17.0-1019"
+UBUNTU_MAINLINE_KERNEL_VERSION="5.15.72"
+UBUNTU_OEM_20.04_KERNEL_VERSION="5.14.0-1049"
```

Comment:
1. Add a table for Key release/Tag
2. Add note in Ubuntu readme/sles readme and main readme in separate pr.
 
2. Ubuntu MAINLINE KERNEL

- 2. This is not the Ubuntu kernel but the Mainline vanilla kernel
- Please add a comment that it's not being tested. Only the build is verified.